openml/extensions/sklearn/extension.py

```diff
@@ -1119,17 +1123,12 @@ def _run_model_on_fold(
             The UNTRAINED model to run. The model instance will be copied and not altered.
         task : OpenMLTask
             The task to run the model on.
+        X_train : array-like
+            Training data for the given repetition and fold.
         rep_no : int
             The repeat of the experiment (0-based; in case of 1 time CV, always 0)
         fold_no : int
             The fold nr of the experiment (0-based; in case of holdout, always 0)
-        sample_no : int
-            In case of learning curves, the index of the subsample (0-based; in case of no
-            learning curve, always 0)
-        add_local_measures : bool
-            Determines whether to calculate a set of measures (i.e., predictive accuracy)
-            locally,
-            to later verify server behaviour.
 
```

Comment:
Looks like `y_train`, `X_test` and `classes` are missing from docstring?

Reply:
Thanks, I forgot to copy them from the parent class.